types/webextension-polyfill/namespaces/tabs.d.ts

```diff
@@ -1195,7 +1180,7 @@ export namespace Tabs {
          * @param details Details of the script to run.
          * @returns Called after all the JavaScript has been executed.
          */
-        executeScript(tabId: number | undefined, details: ExtensionTypes.InjectDetails): Promise<any[]>;
+        executeScript(tabId: number | undefined, details: ExtensionTypes.InjectDetails): Promise<unknown[]>;
```

Comment:
This seems like another change from `any` to `unknown` in an output position.

Reply:
Same reason: only the caller knows what type it's supposed to be.